leaderelection/context_test.go

```diff
@@ -0,0 +1,143 @@
+// +build !race
```

Comment:
is the race not avoidable? this seems like a smell

Reply:
ahhh.. link the issue?

Reply:
I thought I did somewhere, but lemme grep and update if not.